### ISSUE TITLE
Can't interact with the app on Fedora 40

### ISSUE THREAD

**Issue by sebastiano123-c** [nukeop/nuclear#1686]
**Platform:**
Distro: Fedora 40 
Kernel: 6.10.6
DE: swayWM

**Nuclear version:**
v0.6.38
Installation: from flatpak

**Description of the issue:**
The app correctly opens, but I can't interact with it. The only thing I can do is write something in the upper search bar.
But I do not receive any response when I press buttons, or any other element in the GUI. 
The app is simply freezed from the beginning.

When I run the app from the terminal, I receive the following error:
```
$ flatpak run org.js.nuclear.Nuclear

      main › (node:3) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
(Use `nuclear --trace-deprecation ...` to show where the warning was created)
libva error: /usr/lib/x86_64-linux-gnu/dri/intel-vaapi-driver/i965_drv_video.so init failed
(node:91) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
(Use `exe --trace-deprecation ...` to show where the warning was created)
```
Then, when I closed the app, I receive the following message
```
      main › Could not clear discord activity
      main › TypeError: Cannot read property 'write' of null
    at e.exports.send (/app/main/resources/app.asar/dist/webpack:/node_modules/discord-rpc/src/transports/ipc.js:155:17)
    at /app/main/resources/app.asar/dist/webpack:/node_modules/discord-rpc/src/client.js:166:22
    at new Promise (<anonymous>)
    at e.exports.request (/app/main/resources/app.asar/dist/webpack:/node_modules/discord-rpc/src/client.js:164:12)
    at e.exports.clearActivity (/app/main/resources/app.asar/dist/webpack:/node_modules/discord-rpc/src/client.js:550:17)
    at h.clear (/app/main/resources/app.asar/dist/webpack:/@nuclear/main/src/services/discord/index.ts:105:24)
    at App.<anonymous> (/app/main/resources/app.asar/dist/webpack:/@nuclear/main/src/main.ts:116:19)
    at App.emit (events.js:327:22)
```

**Note 1:** I tried with the .rpm package and the AppImage with the very same behaviour.
**Note 2:** When I switch to i3WM the app i working correctly

**Comment by nukeop** [comment]
We never tested Nuclear on Wayland, so it's possible it's due to that.

**Comment by Lucki** [comment]
I am using Nuclear on Wayland with Flatpak just fine.

vaapi points to something with hardware acceleration, I think

**Comment by nukeop** [comment]
Here's what you can try: go to `~/.var/app/org.js.nuclear.Nuclear/config/nuclear/config.json` and add the following entry under `settings`: `"disableGPU": true`.

If Nuclear launched, it should have created a config file there.

**Comment by sebastiano123-c** [comment]
Thank you for getting back to me so quickly.

So I tried
> Here's what you can try: go to `~/.var/app/org.js.nuclear.Nuclear/config/nuclear/config.json` and add the following entry under `settings`: `"disableGPU": true`.
> 
> If Nuclear launched, it should have created a config file there.
but nothing happened.

I have to make an edit. I find the very same problem in VSCodium. The app opens up, it goes smoothly with the keyboard events but the mouse events are not recognized.
I don't know if this is somehow related to Wayland or to SwayWM (I migrated to sway from i3 just the other day). 
Every other program installed that I tested so far works perfectly with the mouse or touchpad events.
